package.json

```diff
@@ -16,14 +16,17 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
+    "pretest": "node pretest.js",
     "test": "mocha --timeout 20000 --require test/init.js",
-    "posttest": "npm run lint"
+    "posttest": "node posttest.js && npm run lint"
```

Comment:
True true.

This would however require to go in depth on each test, find out all the models being created and have a clean up in an after-all hook. I am thinking of doing that change in another PR since all the tests pass (at least the moment).

I have also noticed Juggler imported tests don't clean-up by themselves, so it would require work on that module as well.